backend/src/app/entrypoints/api/routes/copilot_api.py

```
@@ -78,11 +80,22 @@ async def execute_copilot_action(
                     detail="Invalid conversationId format. Must be a UUID.",
                 )
 
-        # Using a generic pattern name for now. This should be defined in your pattern service.
-        # The input variables are passed directly from the request.
+        # Determine pattern_name and input_variables
+        # Priority: 1. Explicit pattern_name/pattern_inputs from request
+        #           2. Default to conversational_chat with message as input
+        current_pattern_name: str
+        current_input_variables: Dict[str, Any]
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)